scripts/tests/twister_blackbox/test_printouts.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Blackbox tests for twister's command line functions
+"""
+
+import importlib
+import mock
+import os
+import pytest
+import sys
+
+
+class TestPrintOuts:
+    ZEPHYR_BASE = os.getenv("ZEPHYR_BASE")
+
+    TESTDATA_1 = [
+        (
+            os.path.join(ZEPHYR_BASE, 'tests', 'kernel'),
+            ['mailbox', 'apic_tsc', 'timer', 'linker_generator', 'libc',
```

Comment:
Do not use live test data, create own, dummy tests with dummy data that is not going to be changed or modified by developers.